app/services/ReviewService.scala

```diff
@@ -34,4 +32,10 @@ class ReviewService @Inject() (reviewConnector: ReviewConnector) {
 
   }
 
+  def approval2iReviewComplete(id: String, status: ApprovalStatus)(implicit ex: ExecutionContext, hc: HeaderCarrier): Future[RequestOutcome[Unit]] = {
+
+    val changeInfo = ApprovalProcessStatusChange("userId", "userName", status)
```

Comment:
what are these strings ?? "userId" and "userName"

Reply:
we don't have the Action that gets these yet - they are in a different PR